spk/salt-minion/Makefile

```diff
@@ -1,41 +1,73 @@
 SPK_NAME = salt-minion
-SPK_VERS = 3001.1
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+SPK_VERS = 3003.2
+else
+SPK_VERS = 3000.9
+endif
```

Comment:
I am intentionally only building last Python 2 supported version for DSM 6.1. In this way I encourage the user to upgrade to DSM 7 (and get latest salt-minion running Python 3).

Reply:
Please, No.
* No added incentive necessary, Synology does a good job there already, people like new and shiny (we received package requests for DSM7 during the preview phase of DSM7 (before the beta launch)).
* This is user hostile / limits older platforms on purpose.
* From memory @SynoCommunity/developers  had already decided not to add any more python 2 packages here.
* Adds more code for no good reason that has to be removed later anyway when DSM 8 comes out or when we can finally remove Python 2 completely.
* If DSM 7 did not make so may changes, we could still support DSM5 as some devices simply can't be upgraded because of raised system requirements in DSM7 (e.g. [your own devices](https://github.com/SynoCommunity/spksrc/pull/4830#issuecomment-910489159))
* It doesn't work yet: https://github.com/SynoCommunity/spksrc/pull/4830#issuecomment-913185626

I hope I convinced you why this is a bad idea.

Reply:
Anyway I'm just worried about the python2 stuff If you still want to limit the version you can do that in the if statements, but please remove the legacy python 2 stuff. [Python2 EOL was 1 year and 8 months ago
(01 Jan 2020)](https://endoflife.date/python)

Reply:
👍 For no more Python 2 stuff!